### DISCUSSION TITLE
Welcome to drawbot Discussions!

### DISCUSSION THREAD

**Discussion by typemytype** [typemytype/drawbot#413] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MauriceMeilleur** [comment]
Hello, everyone—I'm a professor of graphic design at Iowa State University, very interested in type design/typography and generative/computational design. Presently I'm writing a book about/catalogue raisonné of Jurriaan Schrofer's constructed scripts. I use DrawBot/Python as a research/analysis/reconstruction tool, and even though I still feel like a beginner a lot of the time, this year I've started to offer workshops and teach coding with it to students in my classes. And sometimes I just mess around with it.
